src/lib/fees/parameters/index.ts

```diff
@@ -5,14 +5,18 @@ import type { FeeCurveParameters, ParameterModel } from './types'
 export const FEE_CURVE_PARAMETERS: Record<ParameterModel, FeeCurveParameters> = {
   SWAPPER: swapperParameters,
   THORCHAIN_LP: thorchainLpParameters,
+  // THOR asset doesn't have any fee models for now as it's not using curves
+  THORSWAP: {} as FeeCurveParameters,
```

Comment:
We don't need this for now as this swapper doesn't involve any curve

Reply:
We are not consuming this at all, its just to make TS happy

But I liked the zeroed out solution and implemented it in [25dc736](https://github.com/shapeshift/web/pull/8044/commits/25dc7365114443a641a36bd748e76afdffa92ae8) with a little comment